mobify/platform-scaffold
# Add missing Select style to PDP Variants

Ensure that all Select elements are styled correctly. Each `<select>` tag should be wrapped in a `<Field>` (technically a Redux form field) component, with the `pw--has-select` modifier class. For example...

```
<ReduxForm.Field className="pw--has-select" ...>
    <select>{...}</select>
</ReduxForm.Field>
```

## Changes
- Add missing `pw--has-select` class to `select` field

## How to test-drive this PR
- In `app/main.jsx`, comment out the Merlin's Potions connector, and uncomment the SFCC connector
- Run `npm run dev`
- Preview [Merlin's Potions SFCC](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Visit any PDP with product variants
- **Verify** that the variant select dropdowns are styled correctly, and consistently with all other select fields (i.e. as those seen in the PLP/Search sort field, chekcout shipping/payment, and the cart page's estimate shipping modal)
